tests/EmbeddedResources/ResourcesTest.cs

```diff
@@ -31,7 +33,16 @@ public class ResourcesTest {
 		public void Embedded ()
 		{
 #if __TVOS__
-			Assert.Ignore ("This test is disabled on TVOS."); // Randomly crashed on tvOS -> https://github.com/xamarin/maccore/issues/1909
+			bool in_interpreter = false;
+			try {
+				AssemblyName aName = new AssemblyName ("DynamicAssemblyExample");
+				AssemblyBuilder ab = AppDomain.CurrentDomain.DefineDynamicAssembly (aName, AssemblyBuilderAccess.RunAndSave);
```

Comment:
not a big deal but you can just do
```csharp
AppDomain.CurrentDomain.DefineDynamicAssembly (null, , AssemblyBuilderAccess.RunAndSave);
```
since it's default implementation to throw when REPL is not used (which the interpreter reuses)